packages/react-tinacms-editor/src/components/BaseMenubar/index.tsx

```diff
@@ -116,8 +116,8 @@ export const BaseMenubar = ({
         <MenuPortalProvider>
           <MenuContainer onMouseDown={preventProsemirrorFocusLoss}>
             {menus}
-            {plugins?.map(({ MenuItem }) => (
-              <MenuItem mode={mode} editorView={editorView} />
+            {plugins?.map(({ MenuItem }, i) => (
```

Comment:
Could we use `plugin.name` here as opposed to the index?

Reply:
Nice catch. I'm going to merge into a tinacms branch and make that change.